docs/git-experts.md

```diff
@@ -0,0 +1,74 @@
+# Jujutsu for Git experts
+
+People who are proficient with Git often ask what benefit there is to using
+Jujutsu.
+
+## Git is still available to use
+
+Jujutsu repositories are colocated by default, which means they can be used by
+both the `jj` and `git` commands.
+
+If you encounter something you can't do with Jujutsu or is simply easier with
+Git, you have the option of using `git` for that command, then switching back.
+
+## Automatic rebasing makes for less toil
+
+> I already know how to use `git rebase --interactive`.
+
+Rebasing is automatic, so it's one fewer step to take. As soon as you've am.
+
+Git:
+
+- stage
+- commit
+- rebase
+
+Jujutsu:
+
+- squash -i --into
+
+TODO: Show example
+
+TODO: Acknowledge `git rebase --update-refs`
+
+## Undo is easier than searching the reflog
+
+In Git, the reflog tracks the history of each ref (branch). You can see where a
+ref has been over time, and restore it to a previous state with `git reset
+--hard`.
+
+Jujutsu's operation log tracks the complete history of the entire repository,
+not just each ref individually.
+
+If you make a mistake, undoing is as easy as `jj undo`. You don't even have to
+look at Jujutsu's operation log to step back through history.
+
+For more complicated situations, you can view the repository as it existed at
+any point in history. All commands can accept `--at-operation OPERATION_ID` to
+work as though the repository were at that point in time.
+
+TODO: Show example
+
+TODO: show `jj op log -p`, `jj op diff`
+
+## Conflict resolution can be deferred
```

Comment:
this is a big one that we'll have to iterate on, a lot of people want to see something more concrete here. like they don't get the "well you can just do it later" without more details. I had a good example of this somewhere recently and now I've lost it, ugh...

Reply:
I think it's somewhat rare that the "rebase again resolves conflicts" is very useful in practice. I'm not sure it's going to convince many. Perhaps it will be more confusing that it's helpful.

I wonder if this paragraph should be about the conflict algebra (probably without using that term). If it is, I think I would rank 3 things it provides like this:
1. Rebase/merge always succeeds, so we can always rebase descendants
2. We can define the changes in a merge commit well
3. Conflicts resolutions can be deferred

My point is that I think deferring conflict resolution is actually less important as a feature. Just a thought; feel free to ignore.